CMS/templates/partials/subject_selector.html

```diff
@@ -1,24 +1,24 @@
 {% load discover_uni_tags %}
 
 <div class="subject-picker">
-    <div class="course-finder-content__question-selector">
-        <select id="subjectArea">
+    <div class="course-finder-content__question-selector" aria-label="Subject area">
+        <select id="subjectArea" aria-label="Subject area">
```

Comment:
Should these be wrapped in translation tags? if we don't have the translation, ask OFS for it explaining why we need it.